test/index.ts

```diff
@@ -12,6 +13,8 @@ export function run(): Promise<void> {
   const mocha = new Mocha({
     ui: "bdd",
     timeout: 10000,
+    color: true,
+    reporter: "spec",
   });
 
   const testsRoot = path.resolve(__dirname, "..");
```

Comment:
We should create some additional test yamls just to make sure auto-completion and such works with various names - according to the SchemaStore list we support `architect.yml`, `architect.yaml`, `*.architect.yml`, `*.architect.yaml` - so we should have fixtures to make sure all 4 of these styles are recognized as ours/get our extension behavior

Reply:
Good catch, thank you. I'll add those fixures and tests.

Reply:
Added ui tests for `architect.yml, architect.yaml, *.architect.yml, *.architect.yaml`
Added tests for `architect.yml, architect.yaml, *.architect.yml, *.architect.yaml`